src/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -264,12 +264,14 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("text");
 
                     b.Property<string>("MnemonicString")
-                        .IsRequired()
                         .HasColumnType("text");
 
                     b.Property<DateTimeOffset>("UpdateDatetime")
                         .HasColumnType("timestamp with time zone");
 
+                    b.Property<string>("XPUB")
```

Comment:
Is the XPUB not required?

Reply:
Yes because it can be null at the creation of the entity.